packages/dds/tree/src/feature-libraries/forest-summary/format.ts

```diff
@@ -20,13 +20,18 @@ export type ForestFormatVersion = Brand<
 	"ForestFormatVersion"
 >;
 
-export const Format = Type.Object(
-	{
-		version: Type.Literal(ForestFormatVersion.v1),
-		keys: Type.Array(schemaFormatV1.FieldKeySchema),
-		fields: Versioned,
-	},
-	{ additionalProperties: false },
-);
+export const Format = <TVersion extends ForestFormatVersion>(
+	tVersion: TVersion = ForestFormatVersion.v1 as TVersion,
```

Comment:
Added this to change the type of `version` from `number` to `ForestFormatVersion`.
This makes the usage type safe. Caught this problem when debugging tests that were passing a number to this and the compiler didn't complain.